packages/form-field/src/FormField/FormField.tsx

```diff
@@ -62,17 +63,19 @@ export const FormField = forwardRef<HTMLDivElement, FormFieldProps>(
             ref={fwdRef}
             {...rest}
           >
-            <div className={labelTextContainerStyle}>
+            <div className={textContainerStyle}>
               {label && (
-                <Label
-                  data-testid="lg-form_field-label"
-                  className={getFontSize({ baseFontSize, size })}
-                  htmlFor={inputId}
-                  id={labelId}
-                  disabled={disabled}
-                >
-                  {label}
-                </Label>
+                <div className={labelContainerStyle}>
```

Comment:
Could this style be added to the label itself instead of a new div?

Reply:
Updated!